main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -279,6 +307,8 @@ object TokenKind {
 
   case object Minus extends TokenKind
 
+  case object MinusMinus extends TokenKind
```

Comment:
What's this?

Reply:
Unary type operator `--` for CaseDifference
https://github.com/flix/flix/blob/bf14c3ac86d24e9df07b54e8c5257999424e5794/main/src/ca/uwaterloo/flix/language/phase/Parser.scala#L1434

Reply:
Ah-- I guess we need it. At least as long as we support restrictable variants.